editor/filesystem_dock.cpp

```diff
@@ -3321,7 +3321,17 @@ void FileSystemDock::_file_and_folders_fill_popup(PopupMenu *p_popup, const Vect
 		}
 
 		p_popup->add_icon_shortcut(get_editor_theme_icon(SNAME("Filesystem")), ED_GET_SHORTCUT("filesystem_dock/show_in_explorer"), FILE_SHOW_IN_EXPLORER);
-		p_popup->set_item_text(p_popup->get_item_index(FILE_SHOW_IN_EXPLORER), is_directory ? TTR("Open in File Manager") : TTR("Show in File Manager"));
+
+		// Show in file manager string.
+#if defined(WINDOWS_ENABLED)
+		const String item_text = TTR("Reveal in File Explorer");
+#elif defined(MACOS_ENABLED)
+		const String item_text = TTR("Reveal in Finder");
+#else
+		const String item_text = TTR("Reveal in File Manager");
+#endif
```

Comment:
The repeated code needs to be extracted into a method.

Reply:
I fully agree. Do we have already a central file for easy access to translations?

Reply:
You could just add a static method like `FileSystemDock::get_file_manager_option()` or something. `TTR()` is available globally.

Reply:
This should be technically inside the `platforms/` directory. So I'll let this as that, but with a todo to port it to platforms.